auto_nag/scripts/workflow/p2_merge_day.py

```diff
@@ -0,0 +1,63 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from libmozdata import release_calendar as rc
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag import utils
+
+
+class P2MergeDay(BzCleaner):
+    def __init__(self):
+        super(P2MergeDay, self).__init__()
+
+    def must_run(self, date):
+        cal = rc.get_calendar()
+        for c in cal:
+            if date == c['merge']:
+                return True
+        return False
+
+    def description(self):
+        return 'Get bugs with priority P2 and assigned on merge day'
+
+    def name(self):
+        return 'workflow-p2-merge-day'
+
+    def template(self):
+        return 'workflow_p2_merge_day.html'
+
+    def subject(self):
+        return 'Bugs P2 with an assignee on merge day'
+
+    def ignore_bug_summary(self):
+        return False
+
+    def get_bz_params(self, date):
+        params = {
+            'product': 'Core',
+            'resolution': '---',
+            'f1': 'component',
+            'o1': 'casesubstring',
+            'v1': 'Networking',
```

Comment:
Can you move that in the configuration?
For now, it is just Networking but we will extend that to other components.